packages/wonder-blocks-announcer/package.json

```diff
@@ -1,20 +1,28 @@
 {
   "name": "@khanacademy/wonder-blocks-announcer",
+  "description": "Live Region Announcer for Wonder Blocks.",
```

Comment:
I'm open to debate on this.  At one point in Khan/perseus I got tired of the package.json's all defining package.json keys in different orders and missing various common metadata.  

When I did this PR the first time I tried to publish with provenance it failed because that **_requires_** that all package.json's have a `repository.url` that is correct (including casing). 

So I've introduced that script here to do a reformat of all package.json files and make sure they have this common metadata.  

If this is too heavy-handed, I'm happy to back out these changes and just keep the `repository.url` that's needed for this.

Check out the `format-package.json.sh` that is part of this PR for some fun `jq` tricks.